go/vumitools/contact/models.py

```diff
@@ -56,6 +56,13 @@ class ContactNotFoundError(ContactError):
     """Raised when a contact is not found"""
 
 
+def normalize_addr(contact_field, addr):
+    if contact_field == 'msisdn':
+        addr = '+' + addr.lstrip('+')
+    elif contact_field == 'gtalk':
+        addr = addr.partition('/')[0]
+    return (contact_field, addr)
```

Comment:
It seems odd that ``normalize_addr`` returns the ``contact_field`` too. Perhaps it should just return the normalized address?